Snakefile

```diff
@@ -25,143 +25,24 @@ rule k51:
 ## Download sourmash databases & taxonomy files
 ##########################################################
 
-rule download_genbank_bacteria_zip_k21:
-    output: "inputs/sourmash_databases/genbank-2022.03-bacteria-k21.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/6qxfp/download
-    '''
-
-rule download_genbank_bacteria_zip_k31:
-    output: "inputs/sourmash_databases/genbank-2022.03-bacteria-k31.zip"
-    conda: "envs/wget.yml"
-    shell:'''
-    wget -O {output} https://osf.io/9ue5g/download
-    '''
-
-rule download_genbank_bacteria_zip_k51:
-    output: "inputs/sourmash_databases/genbank-2022.03-bacteria-k51.zip"
-    conda: "envs/wget.yml"
```

Comment:
I assume we use this env somewhere else? If not, we can delete the `env` as well.

Reply:
ya i use it lower down good catch though